### ISSUE TITLE
Swift 4 Support

### ISSUE THREAD

**Issue by furick1** [airmap/AirMapSDK-Swift#24]
Does the AirMapSDK support Swift version 4?

**Comment by f2m2rd** [comment]
We will soon suspoirt Swift 4 with s new SDK we are working on.  stay tuned!

**Comment by adolfo** [comment]
@furick1 Swift 4 support is now available via a `2.0.0.beta.1` release tag. Please configure your Podfile as follows to use it:

```ruby
...
pod 'AirMapSDK', :git => 'https://github.com/airmap/AirMapSDK-Swift', :tag => '2.0.0.beta.1'
...

post_install do |installer|
    # List of Pods to use as Swift 3.2
    swift3_targets = ['Lock', 'SwiftMQTT'] 
    
    installer.pods_project.targets.each do |target|
        if swift3_targets.include? target.name
            target.build_configurations.each do |config|
                config.build_settings['SWIFT_VERSION'] = '3.2'
            end
        end
    end
end
```

**Comment by furick1** [comment]
Sweet. Look forward to checking it out. Is there a target timeframe when it will be prime time or does that depend on beta test results?
